presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/SchemaDaoForOld.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptor.metadata;
+
+import org.skife.jdbi.v2.sqlobject.SqlUpdate;
+
+/**
+ * This class is for generating the schema for environment without delta delete
+ */
+public interface SchemaDaoForOld
```

Comment:
Make this an inner class for TestUpgradeMetadata. Call it `TestingLegacySchemaDao`

Reply:
Maybe LegacySchemaDao?😄